src/App.jsx

```diff
@@ -39,7 +39,7 @@ const router = createBrowserRouter([
       },
       {
         path: '/suivi',
-        element: <SuiviPage />,
+        element: <TrackingPage />,
```

Comment:
Can this page be lazy loaded ? in order reduce the bundle size of the app ? With lazy option : 
```suggestion
        lazy: () => import("./pages/TrackingPage"),
```
If you do so, you must remove the static import line 14.